### ISSUE TITLE
Build crashes when using email-templates

### ISSUE THREAD

**Issue by OBrown92** [AnomalyInnovations/serverless-nodejs-starter#61]
Hi,
I'm trying to integrate `email-templates` npm package to my project. I'm using `ejs` as template engine.
When trying to invoke my function locally the webpack build is crashing with the following message:

```sh
➜  serverless-project git:(master) ✗ serverless invoke local --function convert
Serverless: DOTENV: Loading environment variables from .env:
Serverless: 	 - EMAIL_USER
Serverless: 	 - EMAIL_PASS
Serverless: 	 - SLS_DEBUG
Serverless: Load command login
Serverless: Load command logout
Serverless: Load command generate-event
Serverless: Load command test
Serverless: Load command dashboard
Serverless: Load command output
Serverless: Load command output:get
Serverless: Load command output:list
Serverless: Load command param
Serverless: Load command param:get
Serverless: Load command param:list
Serverless: Load command studio
Serverless: Load command dev
Serverless: Invoke invoke:local
Serverless: Invoke webpack:validate
Serverless: Invoke webpack:compile
Serverless: Bundling with Webpack...

ERROR in /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js
Module not found: Error: Can't resolve 'atpl' in '/Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib'
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js 573:51-66
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/index.js
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/email-templates/lib/index.js
 @ /Users/user/Documents/MyProject/serverless-project/lib/mail.js
 @ /Users/user/Documents/MyProject/serverless-project/handler.js

ERROR in /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js
Module not found: Error: Can't resolve 'babel-core' in '/Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib'
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js 1338:50-71 1351:50-71
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/index.js
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/email-templates/lib/index.js
 @ /Users/user/Documents/MyProject/serverless-project/lib/mail.js
 @ /Users/user/Documents/MyProject/serverless-project/handler.js

ERROR in /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js
Module not found: Error: Can't resolve 'bracket-template' in '/Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib'
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js 1196:57-84
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/index.js
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/email-templates/lib/index.js
 @ /Users/user/Documents/MyProject/serverless-project/lib/mail.js
 @ /Users/user/Documents/MyProject/serverless-project/handler.js

ERROR in /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js
Module not found: Error: Can't resolve 'coffee-script' in '/Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib'
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js 1663:15-39
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/index.js
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/email-templates/lib/index.js
 @ /Users/user/Documents/MyProject/serverless-project/lib/mail.js
 @ /Users/user/Documents/MyProject/serverless-project/handler.js

ERROR in /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js
Module not found: Error: Can't resolve 'dot' in '/Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib'
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js 1170:49-63
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/index.js
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/email-templates/lib/index.js
 @ /Users/user/Documents/MyProject/serverless-project/lib/mail.js
 @ /Users/user/Documents/MyProject/serverless-project/handler.js

ERROR in /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js
Module not found: Error: Can't resolve 'dustjs-linkedin' in '/Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib'
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js 490:35-61
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/index.js
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/email-templates/lib/index.js
 @ /Users/user/Documents/MyProject/serverless-project/lib/mail.js
 @ /Users/user/Documents/MyProject/serverless-project/handler.js

ERROR in /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js
Module not found: Error: Can't resolve 'eco' in '/Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib'
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js 664:49-63
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/index.js
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/email-templates/lib/index.js
 @ /Users/user/Documents/MyProject/serverless-project/lib/mail.js
 @ /Users/user/Documents/MyProject/serverless-project/handler.js

ERROR in /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js
Module not found: Error: Can't resolve 'ect' in '/Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib'
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js 1090:16-30 1104:14-28
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/index.js
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/email-templates/lib/index.js
 @ /Users/user/Documents/MyProject/serverless-project/lib/mail.js
 @ /Users/user/Documents/MyProject/serverless-project/handler.js

ERROR in /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js
Module not found: Error: Can't resolve 'ejs' in '/Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib'
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js 642:49-63
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/index.js
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/email-templates/lib/index.js
 @ /Users/user/Documents/MyProject/serverless-project/lib/mail.js
 @ /Users/user/Documents/MyProject/serverless-project/handler.js

ERROR in /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js
Module not found: Error: Can't resolve 'haml-coffee' in '/Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib'
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js 801:71-93
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/index.js
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/email-templates/lib/index.js
 @ /Users/user/Documents/MyProject/serverless-project/lib/mail.js
 @ /Users/user/Documents/MyProject/serverless-project/handler.js

ERROR in /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js
Module not found: Error: Can't resolve 'hamlet' in '/Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib'
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js 753:55-72
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/index.js
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/email-templates/lib/index.js
 @ /Users/user/Documents/MyProject/serverless-project/lib/mail.js
 @ /Users/user/Documents/MyProject/serverless-project/handler.js

ERROR in /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js
Module not found: Error: Can't resolve 'hamljs' in '/Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib'
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js 731:51-68
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/index.js
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/email-templates/lib/index.js
 @ /Users/user/Documents/MyProject/serverless-project/lib/mail.js
 @ /Users/user/Documents/MyProject/serverless-project/handler.js

ERROR in /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js
Module not found: Error: Can't resolve 'handlebars' in '/Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib'
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js 867:63-84
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/index.js
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/email-templates/lib/index.js
 @ /Users/user/Documents/MyProject/serverless-project/lib/mail.js
 @ /Users/user/Documents/MyProject/serverless-project/handler.js

ERROR in /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js
Module not found: Error: Can't resolve 'hogan.js' in '/Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib'
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js 823:53-72
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/index.js
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/email-templates/lib/index.js
 @ /Users/user/Documents/MyProject/serverless-project/lib/mail.js
 @ /Users/user/Documents/MyProject/serverless-project/handler.js

ERROR in /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js
Module not found: Error: Can't resolve 'htmling' in '/Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib'
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js 1324:57-75
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/index.js
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/email-templates/lib/index.js
 @ /Users/user/Documents/MyProject/serverless-project/lib/mail.js
 @ /Users/user/Documents/MyProject/serverless-project/handler.js

ERROR in /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js
Module not found: Error: Can't resolve 'jazz' in '/Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib'
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js 685:51-66
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/index.js
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/email-templates/lib/index.js
 @ /Users/user/Documents/MyProject/serverless-project/lib/mail.js
 @ /Users/user/Documents/MyProject/serverless-project/handler.js

ERROR in /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js
Module not found: Error: Can't resolve 'jqtpl' in '/Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib'
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js 709:53-69
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/index.js
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/email-templates/lib/index.js
 @ /Users/user/Documents/MyProject/serverless-project/lib/mail.js
 @ /Users/user/Documents/MyProject/serverless-project/handler.js

ERROR in /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js
Module not found: Error: Can't resolve 'just' in '/Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib'
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js 1062:17-32 1076:15-30
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/index.js
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/email-templates/lib/index.js
 @ /Users/user/Documents/MyProject/serverless-project/lib/mail.js
 @ /Users/user/Documents/MyProject/serverless-project/handler.js

ERROR in /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js
Module not found: Error: Can't resolve 'liquor' in '/Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib'
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js 595:55-72
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/index.js
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/email-templates/lib/index.js
 @ /Users/user/Documents/MyProject/serverless-project/lib/mail.js
 @ /Users/user/Documents/MyProject/serverless-project/handler.js

ERROR in /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js
Module not found: Error: Can't resolve 'marko' in '/Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib'
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js 1607:53-69 1625:53-69
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/index.js
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/email-templates/lib/index.js
 @ /Users/user/Documents/MyProject/serverless-project/lib/mail.js
 @ /Users/user/Documents/MyProject/serverless-project/handler.js

ERROR in /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js
Module not found: Error: Can't resolve 'mote' in '/Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib'
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js 1122:51-66
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/index.js
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/email-templates/lib/index.js
 @ /Users/user/Documents/MyProject/serverless-project/lib/mail.js
 @ /Users/user/Documents/MyProject/serverless-project/handler.js

ERROR in /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js
Module not found: Error: Can't resolve 'plates' in '/Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib'
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js 1402:55-72
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/index.js
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/email-templates/lib/index.js
 @ /Users/user/Documents/MyProject/serverless-project/lib/mail.js
 @ /Users/user/Documents/MyProject/serverless-project/handler.js

ERROR in /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js
Module not found: Error: Can't resolve 'ractive' in '/Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib'
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js 1218:57-75
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/index.js
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/email-templates/lib/index.js
 @ /Users/user/Documents/MyProject/serverless-project/lib/mail.js
 @ /Users/user/Documents/MyProject/serverless-project/handler.js

ERROR in /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js
Module not found: Error: Can't resolve 'react' in '/Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib'
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js 1438:54-70
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/index.js
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/email-templates/lib/index.js
 @ /Users/user/Documents/MyProject/serverless-project/lib/mail.js
 @ /Users/user/Documents/MyProject/serverless-project/handler.js

ERROR in /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js
Module not found: Error: Can't resolve 'react-dom/server' in '/Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib'
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js 1437:63-90
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/index.js
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/email-templates/lib/index.js
 @ /Users/user/Documents/MyProject/serverless-project/lib/mail.js
 @ /Users/user/Documents/MyProject/serverless-project/handler.js

ERROR in /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js
Module not found: Error: Can't resolve 'slm' in '/Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib'
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js 1590:49-63
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/index.js
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/email-templates/lib/index.js
 @ /Users/user/Documents/MyProject/serverless-project/lib/mail.js
 @ /Users/user/Documents/MyProject/serverless-project/handler.js

ERROR in /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js
Module not found: Error: Can't resolve 'teacup/lib/express' in '/Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib'
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js 1643:55-84
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/index.js
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/email-templates/lib/index.js
 @ /Users/user/Documents/MyProject/serverless-project/lib/mail.js
 @ /Users/user/Documents/MyProject/serverless-project/handler.js

ERROR in /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js
Module not found: Error: Can't resolve 'templayed' in '/Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib'
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js 845:61-81
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/index.js
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/email-templates/lib/index.js
 @ /Users/user/Documents/MyProject/serverless-project/lib/mail.js
 @ /Users/user/Documents/MyProject/serverless-project/handler.js

ERROR in /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js
Module not found: Error: Can't resolve 'toffee' in '/Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib'
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js 1138:55-72 1149:55-72
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/index.js
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/email-templates/lib/index.js
 @ /Users/user/Documents/MyProject/serverless-project/lib/mail.js
 @ /Users/user/Documents/MyProject/serverless-project/handler.js

ERROR in /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js
Module not found: Error: Can't resolve 'twig' in '/Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib'
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js 617:51-66
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/index.js
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/email-templates/lib/index.js
 @ /Users/user/Documents/MyProject/serverless-project/lib/mail.js
 @ /Users/user/Documents/MyProject/serverless-project/handler.js

ERROR in /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js
Module not found: Error: Can't resolve 'vash' in '/Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib'
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js 1553:51-66
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/index.js
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/email-templates/lib/index.js
 @ /Users/user/Documents/MyProject/serverless-project/lib/mail.js
 @ /Users/user/Documents/MyProject/serverless-project/handler.js

ERROR in /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js
Module not found: Error: Can't resolve 'walrus' in '/Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib'
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js 1023:55-72
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/index.js
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/email-templates/lib/index.js
 @ /Users/user/Documents/MyProject/serverless-project/lib/mail.js
 @ /Users/user/Documents/MyProject/serverless-project/handler.js

ERROR in /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js
Module not found: Error: Can't resolve 'whiskers' in '/Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib'
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/lib/consolidate.js 769:59-78 780:59-78
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/consolidate/index.js
 @ /Users/user/Documents/MyProject/serverless-project/node_modules/email-templates/lib/index.js
 @ /Users/user/Documents/MyProject/serverless-project/lib/mail.js
 @ /Users/user/Documents/MyProject/serverless-project/handler.js

  Error --------------------------------------------------

  Error: Webpack compilation error, see above
      at /Users/user/Documents/MyProject/serverless-project/node_modules/serverless-webpack/lib/compile.js:36:17
      at arrayEach (/Users/user/Documents/MyProject/serverless-project/node_modules/lodash/lodash.js:516:11)
      at Function.forEach (/Users/user/Documents/MyProject/serverless-project/node_modules/lodash/lodash.js:9368:14)
      at /Users/user/Documents/MyProject/serverless-project/node_modules/serverless-webpack/lib/compile.js:29:9
      at tryCatcher (/Users/user/Documents/MyProject/serverless-project/node_modules/bluebird/js/release/util.js:16:23)
      at Promise._settlePromiseFromHandler (/Users/user/Documents/MyProject/serverless-project/node_modules/bluebird/js/release/promise.js:547:31)
      at Promise._settlePromise (/Users/user/Documents/MyProject/serverless-project/node_modules/bluebird/js/release/promise.js:604:18)
      at Promise._settlePromise0 (/Users/user/Documents/MyProject/serverless-project/node_modules/bluebird/js/release/promise.js:649:10)
      at Promise._settlePromises (/Users/user/Documents/MyProject/serverless-project/node_modules/bluebird/js/release/promise.js:729:18)
      at Promise._fulfill (/Users/user/Documents/MyProject/serverless-project/node_modules/bluebird/js/release/promise.js:673:18)
      at /Users/user/Documents/MyProject/serverless-project/node_modules/bluebird/js/release/nodeback.js:42:21
      at finalCallback (/Users/user/Documents/MyProject/serverless-project/node_modules/webpack/lib/MultiCompiler.js:254:12)
      at /Users/user/Documents/MyProject/serverless-project/node_modules/webpack/lib/MultiCompiler.js:277:6
      at done (/Users/user/Documents/MyProject/serverless-project/node_modules/neo-async/async.js:2931:13)
      at runCompilers (/Users/user/Documents/MyProject/serverless-project/node_modules/webpack/lib/MultiCompiler.js:181:48)
      at /Users/user/Documents/MyProject/serverless-project/node_modules/webpack/lib/MultiCompiler.js:188:7
      at /Users/user/Documents/MyProject/serverless-project/node_modules/webpack/lib/MultiCompiler.js:270:7
      at finalCallback (/Users/user/Documents/MyProject/serverless-project/node_modules/webpack/lib/Compiler.js:257:39)
      at /Users/user/Documents/MyProject/serverless-project/node_modules/webpack/lib/Compiler.js:306:14
      at AsyncSeriesHook.eval [as callAsync] (eval at create (/Users/user/Documents/MyProject/serverless-project/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:13:1)
      at AsyncSeriesHook.lazyCompileHook (/Users/user/Documents/MyProject/serverless-project/node_modules/tapable/lib/Hook.js:154:20)
      at /Users/user/Documents/MyProject/serverless-project/node_modules/webpack/lib/Compiler.js:304:22
      at Compiler.emitRecords (/Users/user/Documents/MyProject/serverless-project/node_modules/webpack/lib/Compiler.js:499:39)
      at /Users/user/Documents/MyProject/serverless-project/node_modules/webpack/lib/Compiler.js:298:10
      at /Users/user/Documents/MyProject/serverless-project/node_modules/webpack/lib/Compiler.js:485:14
      at _next0 (eval at create (/Users/user/Documents/MyProject/serverless-project/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:14:1)
      at eval (eval at create (/Users/user/Documents/MyProject/serverless-project/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:22:1)
      at /Users/user/Documents/MyProject/serverless-project/node_modules/copy-webpack-plugin/dist/index.js:115:7
      at AsyncSeriesHook.eval [as callAsync] (eval at create (/Users/user/Documents/MyProject/serverless-project/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:18:1)
      at AsyncSeriesHook.lazyCompileHook (/Users/user/Documents/MyProject/serverless-project/node_modules/tapable/lib/Hook.js:154:20)
      at /Users/user/Documents/MyProject/serverless-project/node_modules/webpack/lib/Compiler.js:482:27
      at /Users/user/Documents/MyProject/serverless-project/node_modules/neo-async/async.js:2818:7
      at done (/Users/user/Documents/MyProject/serverless-project/node_modules/neo-async/async.js:3522:9)
      at AsyncSeriesHook.eval [as callAsync] (eval at create (/Users/user/Documents/MyProject/serverless-project/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:4:1)
      at /Users/user/Documents/MyProject/serverless-project/node_modules/webpack/lib/Compiler.js:464:33
      at /snapshot/serverless/node_modules/graceful-fs/graceful-fs.js:152:16
      at /snapshot/serverless/node_modules/graceful-fs/graceful-fs.js:69:14
      at FSReqCallback.oncomplete (fs.js:155:23)

  Get Support --------------------------------------------
     Docs:          docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues
     Issues:        forum.serverless.com

  Your Environment Information ---------------------------
     Operating System:          darwin
     Node Version:              12.18.1
     Framework Version:         1.74.1 (standalone)
     Plugin Version:            3.6.14
     SDK Version:               2.3.1
     Components Version:        2.31.6
```

So what I tried already is to copy my templates with
```yml
custom:
  bundle:
    copyFiles:
      - from: './emails/success/*'
        to: './'
```
to my output and that' working fine. But how can I solve the compiling issue?

**Comment by jayair** [comment]
Hmm that's interesting. Can you create a simple repo for me to test with?

**Comment by OBrown92** [comment]
I created a repo with the solution that worked for me [here](https://github.com/OBrown92/serverless-email-templates).
The problem is, that webpack can not pack some of the packages. I added all the packages from the error response to `externals`. Then another problem occured.
```bash
Error: Cannot find module '../lib/utils.js'
    at Function.webpackEmptyContext [as resolve] (/Users/.../.webpack/convert/webpack:/Users/.../node_modules/uglify-js/tools sync:2:1)
```
which can also be solved by adding `- uglify-js` to the `externals`.
Now it's working.
